dlrs/main/lwc/rollupEditor/rollupEditor.html

```diff
@@ -479,6 +483,23 @@ <h3 class="slds-section__title slds-theme--shade">
             errors={errors.testCodeSeeAllData}
           ></c-rollup-editor-error>
         </lightning-layout-item>
+        <lightning-layout-item
+          size="12"
+          large-device-size="6"
+          class="slds-form-element_stacked"
+        >
+          <lightning-input
```

Comment:
@sharicarlson13 label and help text review, please :)

Reply:
Looks good to me